index.html

```diff
@@ -38,7 +38,7 @@
         alt="background-img"
       />
       <div class="paragraph">
-        <p class="first-text">I’m Alzubair Glad to see you!</p>
+        <h1 class="first-text">I’m Alzubair Glad to see you!</h1>
```

Comment:
Text heading - change the heading from p to h1 to make it more clear and visible